### PR TITLE
remove zetavisor and replace with cosmovisor

### PR DESCRIPTION
**Pull request by billyyang423**
Update Zetavisor and replace with Cosmovisor v1.5.0

### CODE REVIEW

**Review by fadeev — APPROVED** [review]

File: docs/validators/cosmovisor.mdx [hunk 1]
```diff
@@ -95,22 +91,22 @@ Restart=always
 RestartSec=600
 User=zetachain
 LimitNOFILE=262144
-ExecStart=zetavisor start --home /home/zetachain/.zetacored/ --log_format json  --moniker <YOUR_NODE_NAME_HERE>
+ExecStart=Cosmovisor start --home /home/zetachain/.zetacored/ --log_format json  --moniker <YOUR_NODE_NAME_HERE>
```
Comment (CharlieMc0) [inline docs/validators/cosmovisor.mdx:94]: Should be lower case `cosmosvisor`

**Review by CharlieMc0 — CHANGES_REQUESTED** [review]
`cosmovisor` should be lowercase when it's a directory or a command

Make sure `DAEMON_ALLOW_DOWNLOAD_BINARIES=false` in the examples. 

Delete this whole section `https://www.zetachain.com/docs/validators/setting-up-zetavisor/#zetavisor-environment-variables`

File: docs/validators/cosmovisor.mdx [hunk 2]
```diff
@@ -95,22 +91,22 @@ Restart=always
...
 ```
 
 ## Install the `zetacored` Binary
 
 ### Download `zetacored` Binary
 
 Download the latest `zetacored` binary for your system architecture from the following
-links and place it in the `zetavisor/genesis/bin/` directory. This example
+links and place it in the `Cosmovisor/genesis/bin/` directory. This example
```
Comment (CharlieMc0) [inline docs/validators/cosmovisor.mdx:102]: Should be lower case cosmosvisor

File: docs/validators/cosmovisor.mdx [hunk 3]
```diff
@@ -95,22 +91,22 @@ Restart=always
...
 assumes you are using ubuntu 22 on amd64 architecture. You may need to make
 adjustments for your system.
 
 ```bash
 wget https://zetachain-external-files.s3.amazonaws.com/binaries/athens3/latest/zetacored-ubuntu-22-amd64
-mv zetacored-ubuntu-22-amd64 ~/.zetacored/zetavisor/genesis/bin/zetacored
-chmod +x ~/.zetacored/zetavisor/genesis/bin/zetacored
+mv zetacored-ubuntu-22-amd64 ~/.zetacored/Cosmovisor/genesis/bin/zetacored
+chmod +x ~/.zetacored/Cosmovisor/genesis/bin/zetacored
```
Comment (CharlieMc0) [inline docs/validators/cosmovisor.mdx:109]: Should be lower case cosmosvisor

File: docs/validators/cosmovisor.mdx [hunk 4]
```diff
@@ -170,13 +166,13 @@ app.toml file. See the [CosmosSDK Telemetry
...
 
-Breif explanation of the environment variables used by zetavisor.
+Breif explanation of the environment variables used by Cosmovisor.
 
 - `DAEMON_HOME`
 
-  - The location where the zetavisor/ directory is kept that contains the
+  - The location where the Cosmovisor/ directory is kept that contains the
     genesis binary, the upgrade binaries, and any additional auxiliary files
     associated with each binary (e.g. $HOME/.zetacored, etc.).
```
Comment (CharlieMc0) [inline docs/validators/cosmovisor.mdx:177]: Delete this whole section and replace it with 

## More Information About Cosmovisor 
For more detailed information about Cosmosvisor you can visit the [Cosmos Documentation Here](https://docs.cosmos.network/main/build/tooling/cosmovisor)

File: docs/validators/cosmovisor.mdx [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+### Download `zetacored` Binary
+
+Download the latest `zetacored` binary for your system architecture from the following
+links and place it in the `cosmovisor/genesis/bin/` directory. This example
+assumes you are using ubuntu 22 on amd64 architecture. You may need to make
+adjustments for your system.
+
+```bash
+wget https://zetachain-external-files.s3.amazonaws.com/binaries/athens3/latest/zetacored-ubuntu-22-amd64
```
Comment (CharlieMc0) [inline docs/validators/cosmovisor.mdx:91]: Replace this link with https://github.com/zeta-chain/node/releases/download/latest/zetacored-ubuntu-22-amd64

File: docs/validators/cosmovisor.mdx [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+### Create `Cosmovisor` Directories
+
+```bash
+mkdir -p ~/.zetacored/config
+mkdir -p ~/.zetacored/cosmovisor/genesis/bin
+mkdir -p ~/.zetacored/cosmovisor/upgrades
+
+```
+
+### Setup `Cosmovisor` systemd service
```
Comment (CharlieMc0) [inline docs/validators/cosmovisor.mdx:44]: Cosmosvisor should be lower case if it's written as `code` inside ``

**Review by CharlieMc0 — APPROVED** [review]

**Review by CharlieMc0 — APPROVED** [review]

**Review by CharlieMc0 — APPROVED** [review]

**Pull request merged**